diagnostic.md

```diff
@@ -3,67 +3,60 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+#readme > article > p:nth-child(1) > a > img
+#user-content-jquery-dom-diagnostic
+#user-content-prerequisites
+#readme > article > div
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$('li > OL > BODY');
```

Comment:
This will not select anything since it'll attempt to select `<body>` elements that are inside of `<ol>` elements that are inside of `<li>` elements.